<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nopalesGraphics</class>
 <widget class="QMainWindow" name="nopalesGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nopalesGraphics</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="7" colspan="5">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="view1Widget" name="widget1" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1000</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QToolButton" name="toolButton_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="6">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <column>
       <property name="text">
        <string>Scene Collection</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QToolButton" name="toolButton_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QToolButton" name="views">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Views</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QToolButton" name="toolButton_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QToolButton" name="toolButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>1000</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;t&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="transformation">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Transformation&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>16</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>20</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>20</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Escale</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>270</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>290</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_x">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_y">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>150</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>190</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_11">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>250</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_13">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>290</y>
          <width>48</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Colors</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>39</x>
          <y>60</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>31</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>RGB</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>HEXA</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>widget</zorder>
   <zorder>treeWidget</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen_new_Scene"/>
    <addaction name="actionSave_Scene"/>
    <addaction name="actionImport_Scene"/>
    <addaction name="actionExport_Scene"/>
    <addaction name="actionOpen_File_or_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuCopy">
     <property name="title">
      <string>Copy</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModeling">
    <property name="title">
     <string>Modeling</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSculpting">
    <property name="title">
     <string>Sculpting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="title">
     <string>Animation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCurves_Surfaces">
    <property name="title">
     <string>Curves/Surfaces</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPoly_Modeling">
    <property name="title">
     <string>Poly Modeling</string>
    </property>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCreate"/>
   <addaction name="menuSelect"/>
   <addaction name="menuCurves_Surfaces"/>
   <addaction name="menuPoly_Modeling"/>
   <addaction name="menuSculpting"/>
   <addaction name="menuModeling"/>
   <addaction name="menuAnimation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen_File_or_Project">
   <property name="text">
    <string>Open File or Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_new_Scene">
   <property name="text">
    <string>Open new Scene</string>
   </property>
  </action>
  <action name="actionSave_Scene">
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionImport_Scene">
   <property name="text">
    <string>Import Scene</string>
   </property>
  </action>
  <action name="actionExport_Scene">
   <property name="text">
    <string>Export Scene</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>view1Widget</class>
   <extends>QWidget</extends>
   <header>view1widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
